.vscode/settings.json

```diff
@@ -0,0 +1,18 @@
+{
+    // "cmake.generator" can be "Ninja" or "MinGW Makefiles" depending on your environment.
+    "cmake.generator":"Unix Makefiles",
```

Comment:
Shouldn't this be in user settings then? See https://code.visualstudio.com/docs/getstarted/settings

Reply:
I think it's good to have it as a workspace setting since most users will not have it in their global settings.
If the flag is not configured, the default generator then will be used and the project might not compile.

Our workspace requirers the use of Cmake, that is why I thought it's only fair to guide them with that flag.

Cmake [supports multiple generators](https://cmake.org/cmake/help/latest/manual/cmake-generators.7.html), and depending on what you have installed on your system, you can build the project.

For example ninja and unix makefiles support can exist on windows platforms if the user has installed them.

I can also leave it out and users will have to take care of understanding the flags and debugging why it didn't build.